### PR TITLE
Print log lines sent as response to Application Prepare request

### PR DESCRIPTION
**Pull request by ean**
This will show required restart, re-feed and re-index action(s) to the caller.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: client/go/vespa/deploy.go [hunk 1]
```diff
@@ -142,17 +154,22 @@ func Activate(sessionID int64, deployment DeploymentOptions) error {
 		return err
 	}
 	defer response.Body.Close()
+	body, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Got in activate:\n%s\n", string(body))
```
Comment (mpolden) [inline client/go/vespa/deploy.go:161]: Remove?

File: client/go/cmd/deploy.go [hunk 2]
```diff
@@ -177,3 +180,17 @@ func waitForQueryService(sessionOrRunID int64) error {
...
+	for _, entry := range result.LogLines {
+		switch l := entry.Level; l {
+		case "ERROR":
+			fmt.Fprint(os.Stderr, color.Red(l))
+		case "WARNING":
+			fmt.Fprint(os.Stderr, color.Yellow(l))
+		default:
+			fmt.Fprint(os.Stderr, color.Reset(l))
+		}
+		fmt.Fprintf(os.Stderr, " %s", entry.Message)
```
Comment (mpolden) [inline client/go/cmd/deploy.go:194]: All of these should print to `stderr`. Also, shouldn't there be a newline printed here? Or does `entry.Message` always contain one?

File: client/go/vespa/deploy.go (hunk 1 above)
Comment (ean) [reply client/go/vespa/deploy.go]: yes

File: client/go/cmd/deploy.go (hunk 2 above)
Comment (ean) [reply client/go/cmd/deploy.go]: Fixed.
And it looks like entry.Message always contains a newline, so we don't need to append that ourselves.

**Review by mpolden — APPROVED** [review]
👍

**Pull request merged**